src/middlewares/social/social.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * damydata for morking social login on Tacebook Twitter and Google
```

Comment:
@Ramadhan0 `dummy data for mocking social login on Facebook, Twitter and Google`

Please, pay attention to your spellings. Also, can you change the name of this file to better represent its purpose?